tests/tck/features/match/SeekByTag.intVid.feature

```diff
@@ -0,0 +1,113 @@
+Feature: Match seek by tag
+
+  Background: Prepare space
+    Given a graph with space named "nba_int_vid"
+
+  Scenario: seek by empty tag index
+    When executing query:
+      """
+      MATCH (v:bachelor)
+      RETURN id(v) AS vid
+      """
```

Comment:
Is bachelor an empty tag? it seems to have properties with it.

Reply:
Empty tag index not empty tag.